app/views/home/about.html.erb

```diff
@@ -317,21 +317,6 @@
   about six years after we started.
   </p></li>
 
-  <li><p>
-    If you prefer to read Lobsters on a dark background instead of a light one, there are a few options. You could install the
```

Comment:
I feel like we need to update this explanation to talk about when and how the site goes into dark mode, and what variables are worth changing to alter the colors.

Reply:
Good thinking. I'll add a description to the trivia section after I merge this PR.